src/test/java/nl/tistis/pbn/PbnCharTest.java

```diff
@@ -0,0 +1,196 @@
+package nl.tistis.pbn;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PbnCharTest {
+
+    private static final int UNICODE_0_NULL_CHARACTER = 0;
+    private static final int UNICODE_32_SPACE = 32;
+    private static final int UNICODE_33_EXCLAMATION_MARK = 33;
+    private static final int UNICODE_45_HYPHEN = 45;
+    private static final int UNICODE_48_DIGIT_0 = 48;
+    private static final int UNICODE_57_DIGIT_9 = 57;
+    private static final int UNICODE_63_QUESTION_MARK = 63;
+    private static final int UNICODE_65_CAPITAL_A = 65;
+    private static final int UNICODE_90_CAPITAL_Z = 90;
+    private static final int UNICODE_95_UNDERSCORE = 95;
+    private static final int UNICODE_97_SMALL_A = 97;
+    private static final int UNICODE_122_SMALL_Z = 122;
+    private static final int UNICODE_126_TILDE = 126;
+    private static final int UNICODE_160_NON_BREAKING_SPACE = 160;
+    private static final int UNICODE_255_SMALL_Y_WITH_DIAERESIS = 255;
+
+    @Test
+    void isPbnCharReturnsBooleanCheckingIfCharacterIsAValidPortableBridgeNotationStandardCharacter() {
+
+        for (int i = UNICODE_0_NULL_CHARACTER; i <= UNICODE_255_SMALL_Y_WITH_DIAERESIS; i++) {
```

Comment:
I'm wondering if there's a way to do this via parameterized tests?

https://blog.codefx.org/libraries/junit-5-parameterized-tests/

Reply:
bet there is. https://www.baeldung.com/parameterized-tests-junit-5